### PR TITLE
6.1 Fix pep8

### PR DESCRIPTION
**Pull request by bwrsandman**

### CODE REVIEW

**Comment by bwrsandman** [comment]
Build gives a false positive!

```
except_orm: (u'Error', u"You try to install module 'hr_interview' that depends on module 'smtpclient'.\nBut the latter module is not available in your system.")
```

File: hr_change_password/wizard/change_passwd_wizard.py [hunk 1]
```diff
@@ -20,8 +20,9 @@
 #
 #
 
-from osv import osv, fields
+from osv import osv
 import pooler
+from tools.translate import _
 
 
 class change_passwd(osv.osv_memory):
```
Comment (pedrobaeza) [inline hr_change_password/wizard/change_passwd_wizard.py:28]: Better orm.TransientModel?

File: hr_contract_timesheet/hr_contract_timesheet.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
 #
 
 import time
-from osv import fields, osv
+from osv import osv
 
 
 class hr_timesheet(osv.osv):
```
Comment (pedrobaeza) [inline hr_contract_timesheet/hr_contract_timesheet.py:26]: orm.Model

File: hr_holidays_request/hr_holidays_request.py [hunk 3]
```diff
@@ -18,10 +18,7 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 #
-from mx import DateTime
 import time
-import pooler
-import netsvc
 import datetime
 import calendar
 from osv import fields, osv
```
Comment (pedrobaeza) [inline hr_holidays_request/hr_holidays_request.py:24]: from openerp.osv ...

File: hr_holidays_evaluation/hr_holidays_evaluation.py [hunk 4]
```diff
@@ -21,11 +21,9 @@
 
 from mx import DateTime
 import time
-import datetime
 
 from osv import osv, fields
```
Comment (pedrobaeza) [inline hr_holidays_evaluation/hr_holidays_evaluation.py:25]: from openerp.osv ...

File: hr_contract_timesheet/hr_contract_timesheet.py [hunk 5]
```diff
@@ -20,7 +20,7 @@
 #
 
 import time
-from osv import fields, osv
+from osv import osv
```
Comment (pedrobaeza) [inline hr_contract_timesheet/hr_contract_timesheet.py:23]: from openerp.osv...

File: hr_change_password/wizard/change_passwd_wizard.py [hunk 6]
```diff
@@ -20,8 +20,9 @@
 #
 #
 
-from osv import osv, fields
+from osv import osv
```
Comment (pedrobaeza) [inline hr_change_password/wizard/change_passwd_wizard.py:23]: from openerp.osv...

File: hr_change_password/wizard/change_passwd_wizard.py [hunk 7]
```diff
@@ -20,8 +20,9 @@
...
 import pooler
+from tools.translate import _
```
Comment (pedrobaeza) [inline hr_change_password/wizard/change_passwd_wizard.py:25]: from openerp.tools...

File: hr_experience_analytic/hr_experience_analytic.py [hunk 8]
```diff
@@ -24,7 +24,12 @@
 class hr_professional(osv.osv):
```
Comment (pedrobaeza) [inline hr_experience_analytic/hr_experience_analytic.py:24]: orm.Model

File: hr_holidays_evaluation/hr_holidays_evaluation.py [hunk 9]
```diff
@@ -21,11 +21,9 @@
...
 from tools.translate import _
-import pooler
 
 
 class hr_holidays(osv.osv):
```
Comment (pedrobaeza) [inline hr_holidays_evaluation/hr_holidays_evaluation.py:29]: orm.Model

File: hr_holidays_type/hr_holidays_type.py [hunk 10]
```diff
@@ -229,28 +256,36 @@ def holidays_validate(self, cr, uid, ids, *args):
...
+                # if record.category_id and not record.holiday_status_id.limit:
                 if record.category_id:
                     for j in record.holiday_line:
                         leave_asked = i.number_of_days
                         leaves_rest = self.pool.get('hr.holidays.status').get_days_cat(
-                            cr, uid, [j.holiday_status_id.id], record.category_id.id, False)[j.holiday_status_id.id]['remaining_leaves']
+                            cr, uid, [j.holiday_status_id.id], record.category_id.id, False
+                        )[j.holiday_status_id.id]['remaining_leaves']
                         if leaves_rest < leave_asked and not i.holiday_status_id.limit:
                             raise osv.except_osv(
```
Comment (pedrobaeza) [inline hr_holidays_type/hr_holidays_type.py:284]: orm.except_orm

File: hr_interview/hr_interview.py [hunk 11]
```diff
@@ -21,7 +21,6 @@
 #
 
 from osv import fields, osv
```
Comment (pedrobaeza) [inline hr_interview/hr_interview.py:23]: from openerp.osv...

File: hr_interview/hr_interview.py [hunk 12]
```diff
@@ -227,4 +246,3 @@ class hr_interview_log(osv.osv):
         'user_id': fields.many2one("res.users", "User Name")
     }
 hr_interview_log()
```
Comment (pedrobaeza) [inline hr_interview/hr_interview.py:248]: It's not needed

File: hr_interview/wizard/wizard_send_interview_mail.py [hunk 13]
```diff
@@ -22,10 +22,7 @@
 
 from osv import osv
```
Comment (pedrobaeza) [inline hr_interview/wizard/wizard_send_interview_mail.py:23]: from openerp.osv...

File: hr_payroll_declare/hr_payroll_declare.py [hunk 14]
```diff
@@ -22,7 +22,6 @@
 #
 import time
 from datetime import datetime
-from datetime import timedelta
 
 from osv import osv, fields
```
Comment (pedrobaeza) [inline hr_payroll_declare/hr_payroll_declare.py:26]: from openerp.osv

File: hr_performance/hr_performance.py [hunk 15]
```diff
@@ -19,9 +19,8 @@
 #
 #
 import datetime
-import time
-from mx.DateTime import *
 from osv import fields,  osv
```
Comment (pedrobaeza) [inline hr_performance/hr_performance.py:22]: from openerp.osv...

File: hr_performance/hr_performance.py [hunk 16]
```diff
@@ -245,7 +244,8 @@ def write(self, cr, uid, ids, vals, context=None):
             if vals.__contains__('obtained_marks'):
                 if vals['obtained_marks'] > self_obj.total_marks or vals['obtained_marks'] < 0:
                     raise osv.except_osv(
```
Comment (pedrobaeza) [inline hr_performance/hr_performance.py:246]: orm.except_orm

File: hr_planned_leaves/report/hr_planned_leaves.py [hunk 17]
```diff
@@ -22,12 +22,9 @@
 
 import pooler
 from report.interface import report_rml
-from tools import to_xml
 import tools
 import datetime
 import time
-from osv import fields, osv
-from report.interface import toxml
 from tools.translate import _
```
Comment (pedrobaeza) [inline hr_planned_leaves/report/hr_planned_leaves.py:28]: from openerp.tools...

File: hr_planned_leaves/wizard/hr_planned_leaves_wizard.py [hunk 18]
```diff
@@ -19,7 +19,7 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 #
-import time
+
 import datetime
 import calendar
 from osv import osv, fields
```
Comment (pedrobaeza) [inline hr_planned_leaves/wizard/hr_planned_leaves_wizard.py:25]: from openerp.osv...

**Comment by pedrobaeza** [comment]
I have made some conventions comments, but nothing about PEP8.

Regards.

**Comment by bwrsandman** [comment]
@pedrobaeza this is a 6.1 branch, so I didn't take as special care to fix it up.

**Comment by pedrobaeza** [comment]
OK, you're right. It can be also, but let's concentrate on the others.

Thanks for the PEP8 ;)

:+1:

**Comment by rvalyi** [comment]
:+1:  agree for this improvement, let's be perfect on v8, not 6.1

**Pull request merged**